proto/auth/auth.proto

```diff
@@ -18,9 +19,14 @@ message AuthContainer {
   AuthBody protectedPayload = 1;      // envelope body, a marshalled protobuf data or it can be null
   // if the length of senderCertHash received is not N bytes, as described in hashAlgorithm, then the protobuf
   // message either is not AuthContainer type, or is corrupted. Otherwise, the
-  // receiver may not have the sender's signing certificate
+  // receiver may not have the sender's signing certificate.
+  // If `cipherData` is defined and valid, then `protectedPayload` is null and actual payload is
+  // defined in the `cipherData.cipherData` member. Once the `CipherBlock` is decrypted,
+  // `protectedPayload` is defined by the auth stack.
```

Comment:
I know what you're trying to say but I don't know who will understand it. Would it make sense to re-word to say e..g., Once the cipherblock is decrypted the result of the decryption is fed into the authentication as if it had been in AuthBody.

That raises the question whether the encrypted payload an AuthBody or the embeeded bytes (Authbody.payload). Makes sense to be specific about that.

Reply:
Will re-word it. Let's see how it goes.